state/stateConsensus.go

```diff
@@ -608,6 +608,11 @@ func (s *State) ReviewHolding() {
 }
 
 func (s *State) MoveStateToHeight(dbheight uint32, newMinute int) {
+
+	if newMinute == 0 {
+		s.CheckForIDChange(dbheight)
+	}
+
```

Comment:
TODO: moove below log & make check more comprehensive - need to check that height is changing and new min is 0

Reply:
check that LLeader hight is changing